lumimqtt/lumimqtt.py

```diff
@@ -301,23 +299,47 @@ async def _periodic_publish(self, period=1):
                             value=value,
                             last_sent=datetime.now(),
                         )
-                        await self._client.publish(
-                            aio_mqtt.PublishableMessage(
-                                topic_name=self._get_topic(sensor.topic),
-                                payload=value,
-                                qos=aio_mqtt.QOSLevel.QOS_1,
-                                retain=self._sensor_retain,
-                            ),
-                        )
-                except (
-                    aio_mqtt.ConnectionClosedError,
-                    aio_mqtt.ServerDiedError,
-                ) as e:
-                    logger.error("Connection closed", exc_info=e)
-                    await self._client.wait_for_connect()
-                    continue
+                        await self._publish_sensor(sensor, value)
+
+                for light in self.lights:
+                    now = datetime.now()
+                    should_send = (
+                        self._light_last_sent is None or
+                        (now - self._light_last_sent).seconds >= self._light_tele_period
+                    )
+                    if should_send:
+                        await self._publish_light(light)
+                        self._light_last_sent = datetime.now()
+            except (
+                aio_mqtt.ConnectionClosedError,
+                aio_mqtt.ServerDiedError,
+            ) as e:
+                logger.error("Connection closed", exc_info=e)
+                await self._client.wait_for_connect()
+                continue
+
             await aio.sleep(period)
 
+    async def _publish_sensor(self, sensor: Sensor, value=False):
```

Comment:
Change the default value to None, please. It is not a boolean value. 
Also, zero value should be allowed.